metadata/src/metadata/convert.js

```diff
@@ -0,0 +1,21 @@
+const { TypeRegistry } = require("@polkadot/types");
+const { hexStripPrefix } = require("@polkadot/util");
+const path = require("path");
+const rawMetadata = require(path.join(__dirname, "../../metadata.json"));
+
+
+function metadataToHex() {
```

Comment:
IMO it would be better and more robust if we pass the metadata json file as a parameter to this function. This way this logic will not be attached to "where the metadata" is placed

Reply:
I guess that this will come naturally when we do the second part of the metadata (if you think that is the case as-well, we can postpone it for the next PR)